ramls/po_line.raml

```diff
@@ -44,6 +44,6 @@ resourceTypes:
         type: UUID
     type:
       collection-item:
-        exampleItem: !include acq-models/mod-orders-storage/examples/po_line_get.sample
-        schema: po_line
+        exampleItem: !include acq-models/mod-orders-storage/examples/po_line_post.sample
```

Comment:
po_line_get.sample makes more sense here because we will not have a POST end point for /{id}

Reply:
revert this line change... should still point to get sample

Reply:
Changed

Reply:
Still points to the post sample

Reply:
Updated.

Reply:
It still points to the post on line 47 John. I think you are looking at line 31